main.yml

```diff
@@ -194,9 +198,6 @@ landing:
 
 migrations:
   title: Migrations
-  api: 
```

Comment:
@ryandillinfelton Migrations is top level nav with children right? Because I don't think that it has api (migration-analytics on other hand has xavier api).

Reply:
Good catch. Bundles shouldn't have an associated API afaik